src/daos/build/install-single-host-dgw.sh

```diff
@@ -177,6 +179,40 @@ function build_daos()
     setup_hugepages
 }
 
+build_ccache()
+{
+    sudo mkdir -p $CCACHE_BUILD_PATH
+    sudo chmod 777 $CCACHE_BUILD_PATH
+	pushd $CCACHE_BUILD_PATH
+	wget https://github.com/ccache/ccache/releases/download/v4.6.1/ccache-4.6.1.tar.xz
+	tar -xf ccache-4.6.1.tar.xz
+	cd ccache-4.6.1/
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc  -DZSTD_FROM_INTERNET=ON -DHIREDIS_FROM_INTERNET=ON ..
+	make
+	sudo make install
+    popd
+    sudo mkdir -p $CCACHE_PATH
+    sudo chmod -R 777 $CCACHE_PATH
+}
+
+function starlet_workaround()
+{
+    # as a workaround for xmlstarlet being removed from the el8 repo, we've stored the Seagate repo
+    set +e
+    sudo dnf provides xmlstarlet
+    local dnf_result=$?
+    set -e
+    if [[ ! $dnf_result == 0 ]]; then
+        starlet_manually_installed=true
+        sed -i "/xmlstarlet/d" ceph.spec.in
```

Comment:
Is this strictly necessary?

Reply:
Yes.  This is the repo issue where xmlstarlet was removed from the @system repo.  Without it, it will not build outside of the Seagate network.  I would think Intel may want to build it themselves at some point in time.

Reply:
Rather than going though pip and editing Ceph's stuff, could we just build the RPM here if it's not available in the repos?

Reply:
Its an edit followed by a restore, so there are no actual changes.  Feel free to update it to build it.  Just make sure you test it from outside of Seagate's network.
It will only run that code if xmlstarlet is not available from the dnf command